### DISCUSSION TITLE
How to calculate point size with camera distance？

### DISCUSSION THREAD

**Discussion by August1996** [google/filament#6318] [Q&A] [answered]
```
void materialVertex(inout MaterialVertexInputs material) {
    gl_PointSize = 100.0;
}
```

No matter where the camera eye is, the point size is 100, how can i calculate the scale of point size with camera eye distance?

@romainguy @bejado @pixelflinger

**Comment by pixelflinger** [comment, marked as answer]
It's basically something like

size = size_at_1_unit / z

**Reply by August1996** [reply]
@pixelflinger how to know size_at_1_unit?
